[2022-02-19 13:47:43,708] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: demo_email.Send_Email manual__2022-02-19T13:47:38.570484+00:00 [queued]>
[2022-02-19 13:47:43,719] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: demo_email.Send_Email manual__2022-02-19T13:47:38.570484+00:00 [queued]>
[2022-02-19 13:47:43,719] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 13:47:43,719] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-19 13:47:43,719] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 13:47:43,755] {taskinstance.py:1259} INFO - Executing <Task(EmailOperator): Send_Email> on 2022-02-19 13:47:38.570484+00:00
[2022-02-19 13:47:43,759] {standard_task_runner.py:52} INFO - Started process 440 to run task
[2022-02-19 13:47:43,781] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'demo_email', 'Send_Email', 'manual__2022-02-19T13:47:38.570484+00:00', '--job-id', '360', '--raw', '--subdir', 'DAGS_FOLDER/pandastest.py', '--cfg-path', '/tmp/tmp6htf98mx', '--error-file', '/tmp/tmps71f89hu']
[2022-02-19 13:47:43,782] {standard_task_runner.py:77} INFO - Job 360: Subtask Send_Email
[2022-02-19 13:47:43,889] {logging_mixin.py:109} INFO - Running <TaskInstance: demo_email.Send_Email manual__2022-02-19T13:47:38.570484+00:00 [running]> on host d18a6d5a39bc
[2022-02-19 13:47:43,977] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=demo_email
AIRFLOW_CTX_TASK_ID=Send_Email
AIRFLOW_CTX_EXECUTION_DATE=2022-02-19T13:47:38.570484+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-19T13:47:38.570484+00:00
[2022-02-19 13:47:44,034] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/email.py:102 PendingDeprecationWarning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
[2022-02-19 13:47:44,035] {configuration.py:381} WARNING - section/key [smtp/smtp_user] not found in config
[2022-02-19 13:47:44,035] {email.py:208} INFO - Email alerting: attempt 1
[2022-02-19 13:47:44,077] {email.py:220} INFO - Sent an alert email to ['test@mailhog.local']
[2022-02-19 13:47:44,184] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=demo_email, task_id=Send_Email, execution_date=20220219T134738, start_date=20220219T134743, end_date=20220219T134744
[2022-02-19 13:47:44,329] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-19 13:47:44,378] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
